dotnet/Data/WishListRepository.cs

```diff
@@ -82,34 +80,40 @@ public async Task<bool> SaveWishList(IList<ListItem> listItems, string shopperId
             };
 
             var jsonSerializedListItems = JsonConvert.SerializeObject(wishListWrapper);
-            var request = new HttpRequestMessage
+            bool IsSuccessStatusCode = false;
```

Comment:
Ideally, local variables should begin with a lowercase character: isSuccessStatusCode